src/styles/navigation.scss

```diff
@@ -0,0 +1,55 @@
+.navigation {
+  margin-right: 50px;
```

Comment:
A block must not be positioned or depend on other blocks. Don't create margins on main block